### ISSUE TITLE
low pitched sounds

### ISSUE THREAD

**Issue by Shivaxi** [omf2097/openomf#352]
A lot of the in game sounds are super low pitched far more than they're supposed to be, all the hitting sound effects of punching and kicking, swinging, landing and jumping, even hazard sounds like the planes on the desert arena and pretty much any move effects like leap and concussion cannon from jaguar.  The music is fine and the announcer sounds fine when saying "round 1, fight" etc.  The menu sounds are also missing (moving between selections).  Intro sounds with the lightning is fine.

Also great work so far just wanted to say, my childhood game right here, played this all the time and even still did from time to time running it through dosbox, first played this when I was merely 3 years old the year it came out, bigups guys.

**Comment by katajakasa** [comment]
Yeah, we're aware of the sound issue. This happens because we're not (yet) 100% sure on how we should handle the sound effect tags. Current implementation is more or less a guess on how they work, and as a result we get oddities in playback. Something to be improved in the future, to be sure.

Also, thanks :)

**Comment by Shivaxi** [comment]
If I may inquire, it sounds very much like the exact issue you'd get when playing back sounds at the wrong sample rate.  Could it perhaps be related to this?  I'm betting a lot of the sounds in OMF were at 22050 or even 11025 possibly, and maybe OpenOMF is trying to play em back at 44100 or 48000hz?

**Comment by katajakasa** [comment]
The audio samples are mono 8bit 8000Hz, and that's the default playback samplerate. However, OMF modifies the samplerate and panning dynamically depending on animation. Normal hit sound effectds might play at normal samplarate, while other hits might play it with lower samplerate to signify eg. stronger punch.

* Explanation of animation string tags http://wiki.omf2097.com/doku.php?id=openomf:animation_string#tags 
* Our tool for looking up tag usage https://katajakasa.fi/projects/openomf/strings/

**Comment by Vagabond** [comment]
Closing as largely fixed.
